### DISCUSSION TITLE
Nube Mistake: Page Not Opening in app but target _blank

### DISCUSSION THREAD

**Discussion by taziki** [quasarframework/quasar#10255] [CLI - Capacitor / Cordova modes]
Hi Team, 

Am testing this framework but I think I fiddled when I am creating primary links they are opening my app pages in new tab and not in app. 

<template>
  <q-layout view="lHh Lpr lFf">
    <q-header elevated>
      <q-toolbar>
        <q-btn
          flat
          dense
          round
          icon="menu"
          aria-label="Menu"
          @click="toggleLeftDrawer"
        />

        <q-toolbar-title>
         Vhembe TVET College
        </q-toolbar-title>

        <div>Geared For Success.</div>
      </q-toolbar>
    </q-header>

          <q-footer elevated>
        <q-toolbar>
          <q-toolbar-title>Footer</q-toolbar-title>
        </q-toolbar>
      </q-footer>
<q-drawer
        v-model="leftDrawerOpen"
        show-if-above
        :width="250"
        :breakpoint="600"
      >
        <q-scroll-area style="height: calc(100% - 150px); margin-top: 150px; border-right: 1px solid #ddd">
          <q-list padding>

        <q-item-label
          header
          class="text-grey-8"
        >
          Essential Links
        </q-item-label>

        <EssentialLink
          v-for="link in essentialLinks"
          :key="link.title"
          v-bind="link"
        />   

      <q-item 
      to="/about"
      clickable 
      v-ripple>
        <q-item-section avatar>
          <q-avatar>
            <img src="icons/group.png">
          </q-avatar>
        </q-item-section>
        <q-item-section>About Us</q-item-section>
      </q-item>

            <q-item clickable v-ripple to="/OurCampuses">
        <q-item-section avatar>
          <q-avatar>
            <img src="icons/campus.png">
          </q-avatar>
        </q-item-section>
        <q-item-section>Our Campuses</q-item-section>
      </q-item>

            <q-item clickable v-ripple to="/OnlineApplication">
        <q-item-section avatar>
          <q-avatar>
            <img src="icons/online-course.png">
          </q-avatar>
        </q-item-section>
        <q-item-section>Our Courses</q-item-section>
      </q-item>

      <q-separator />

      <q-item clickable v-ripple to="/OnlineApplication">
        <q-item-section avatar>
          <q-avatar>
            <img src="icons/web-browser.png">
          </q-avatar>
        </q-item-section>
        <q-item-section>Apply Online</q-item-section>
      </q-item>

      <q-item clickable v-ripple to="/events">
        <q-item-section avatar>
          <q-avatar square>
            <img src="icons/planner.png">
          </q-avatar>
        </q-item-section>
        <q-item-section>Events</q-item-section>
      </q-item>

      <q-item clickable v-ripple to="/help">
        <q-item-section avatar>
          <q-avatar rounded>
            <img src="icons/contact.png">
          </q-avatar>
        </q-item-section>
        <q-item-section>Contact Us</q-item-section>
      </q-item>

      <q-separator />

      <q-item clickable v-ripple to="/">
        <q-item-section avatar>
          <q-avatar rounded>
            <img src="icons/worldwide.png">
          </q-avatar>
        </q-item-section>
        <q-item-section>Our Main Website</q-item-section>
      </q-item>

      <q-item clickable v-ripple to="externalsite" exact>
        <q-item-section avatar>
        <q-avatar rounded>
        <img src="icons/tweet.png">
        </q-avatar>
        </q-item-section>
        <q-item-section>Twitter</q-item-section>
      </q-item>

      <q-item clickable v-ripple to="/help">
        <q-item-section avatar>
        <q-avatar rounded>
        <img src="icons/facebook.png">
        </q-avatar>
        </q-item-section>
        <q-item-section>Facebook</q-item-section>
      </q-item>

 <q-separator>

 </q-separator>
       <q-item clickable v-ripple to="/help">
        <q-item-section avatar>
          <q-avatar rounded>
          <img src="icons/document.png">
          </q-avatar>
        </q-item-section>
        <q-item-section>Student & Staff Zone</q-item-section>
      </q-item>

         </q-list>
        </q-scroll-area>

        <q-img class="absolute-top" src="/icons/bg_top.png" style="height: 150px">
          <div class="absolute-bottom bg-transparent">
            <q-avatar size="84px" class="q-mb-sm" >
            <img src="/icons/logo_avartar.jpg">
            </q-avatar>
            <div class="text-weight-bold">Vhembe College</div>
            <div>@VhembeTVETCol </div>
          </div>
        </q-img>
      </q-drawer>

      <q-page-container>
        <keep-alive>
            <router-view />
        </keep-alive>

      <q-page padding>
      </q-page>
      </q-page-container>
  </q-layout>
</template>

<script>
import EssentialLink from 'components/EssentialLink.vue'

const linksList = 

[

  
];

import { defineComponent, ref } from 'vue'

export default defineComponent({
  name: 'MainLayout',

  components: {
    EssentialLink
  },

  setup () {
    const leftDrawerOpen = ref(false)

    return {
      essentialLinks: linksList,
      leftDrawerOpen,
      toggleLeftDrawer () {
        leftDrawerOpen.value = !leftDrawerOpen.value
      }
    }
  }
})
</script>

**Comment by ontwikkelfabriek** [comment]
@taziki 

Please for better support, instead of posting long code listings, create a codepen.io to demo your problem.

**Reply by taziki** [reply]
thanks.
